user-interface/src/main/java/life/qbic/datamanager/views/projects/project/samples/BatchDetailsComponent.java

```diff
@@ -119,6 +121,7 @@ private void createBatchGrid() {
         .setHeader("Action");
     batchGrid.addThemeVariants(GridVariant.LUMO_COMPACT);
     batchGrid.addClassName("batch-grid");
+    batchGrid.setAllRowsVisible(true);
```

Comment:
why is this necessary?

Reply:
This enables the grid to be dynamic in size dependent on the number of batches. 
This is intended to decrease the space the batch component will take on the sample page since we can expect a less number of batches for an experiment normally than is the default height of 400px
https://vaadin.com/docs/latest/components/grid#dynamic-height